### PR TITLE
docker image fixes for topic/schema scraper

### PR DESCRIPTION
**Pull request by arov00**
- update jdk version in jvm image
- make built executable actually executable in native images

### CODE REVIEW

File: kafka-scraper/src/main/docker/Dockerfile.native [hunk 1]
```diff
@@ -24,4 +24,5 @@ COPY --chown=1001:root target/*-runner /work/application
 EXPOSE 8080
 USER 1001
 
+RUN chmod +x /work/application
```
Comment (arov00) [inline kafka-scraper/src/main/docker/Dockerfile.native:27]: To give a bit of context on why this is needed: Currently, running the native image fails with the following error:

```
docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "./application": permission denied: unknown.
```

Comment (gaetancollaud) [reply kafka-scraper/src/main/docker/Dockerfile.native:27]: I think this is a github action issue, not a docker issue. It's when we upload/download artefact. Somehoe the permissions are lost.
I would rather have a `chmod +x` in the github actions directly (just after the download).

Comment (gaetancollaud) [reply kafka-scraper/src/main/docker/Dockerfile.native:27]: link: https://github.com/actions/download-artifact/issues/14

Comment (gaetancollaud) [reply kafka-scraper/src/main/docker/Dockerfile.native:27]: A simple solution would be to not split `build` and `publish` and to put the condition on the steps directly

Comment (arov00) [reply kafka-scraper/src/main/docker/Dockerfile.native]: Good idea. I've updated the workflow and removed the `chmod` call again.

File: .github/workflows/build-scraper.yaml [hunk 2]
```diff
@@ -29,31 +29,15 @@ jobs:
         working-directory: kafka-scraper/
         run: mvn --batch-mode package -DskipTests -Pnative
 
-      - name: Upload artifacts
```
Comment (gaetancollaud) [inline .github/workflows/build-scraper.yaml:32]: could you do this for the aggregator too plz ?

Comment (gaetancollaud) [inline .github/workflows/build-scraper.yaml:32]: could you do this for the aggregator too plz ?

**Review by gaetancollaud — APPROVED** [review]

**Pull request merged**